unlock-app/src/theme/globalStyle.js

```diff
@@ -31,34 +31,6 @@ export const globalStyle = `
     --alwaysontop: 100000;
   }
 
-  * {
-    box-sizing: border-box;
-  }
-
-  body {
-    font-family: IBM Plex Sans, Helvetica Neue, Arial, sans-serif;
-    color: var(--grey);
-  }
-
-  h1 {
-    font-size: 24px;
-    font-weight: 500;
-  }
-
-  h2 {
-    font-size: 15px;
-    font-weight: 500;
-  }
-
-  a {
-    text-decoration: none;
-    color: var(--link);
-  }
-
-  a:visited {
-    color: var(--link);
-  }
-
```

Comment:
This gets rid of these global styles because they will interfere with new design (different font, decoration, sizing, etc).